content/rust/docker/rust-docker-arm.md

```diff
@@ -12,3 +12,365 @@ tags:
 - docker
 - arm
 ---
+
+This guide will walk you through the steps required to build and publish arm64 Docker images for Rust projects. This guide assumes you have a basic understanding of Rust and Docker, and that you have Rust and Docker installed.
+
+# Setup a Rust Project
+
+First we setup an example project. To do this we can use `cargo`:
```

Comment:
Replace with

```
Set up an example project. We use `cargo`
```

and then make a link to `cargo` to the project, just in case someone is new enough to Rust that they need to look it up.